### ISSUE TITLE
Some questions about main.py

### ISSUE THREAD

**Issue by zhang0391** [n0obcoder/NIH-Chest-X-Rays-Multi-Label-Image-Classification-In-Pytorch#3]
How should I modify the line 30 in main.py?
parser.add_argument('--data_path', type = str, default = 'NIH Chest X-rays', help = 'This is the path of the training data')

**Comment by n0obcoder** [comment]
whats the modification that you want to make ?

**Comment by zhang0391** [comment]
I want to know how should modify default = 'NIH Chest X-rays', what does NIH Chest X-rays cotains?

**Comment by n0obcoder** [comment]
thats a 40 GB dataset of Chest X-Ray images
You can find that [here](https://www.kaggle.com/nih-chest-xrays/data)

**Comment by zhang0391** [comment]
So can I put the entire NIH Chest X-rays folder path  directly?

**Comment by n0obcoder** [comment]
You just have to download the NIH dataset in the root directory of this repository. The code shall take care of everything else ! :D

**Comment by zhang0391** [comment]
OK, I know that and thank you very much!

**Comment by n0obcoder** [comment]
you're welcome @zhang0391 ! : ) i am glad i could help
